sagemaker-python-sdk/tensorflow_keras_cifar10/tensorflow_keras_CIFAR10.ipynb

```diff
@@ -0,0 +1,213 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using Keras modules with TensorFlow\n",
+    "\n",
+    "This notebook shows how to use Keras models with SageMaker using the TensorFlow container.\n",
+    "The model used for this notebook is a simple deep CNN model, trained with the CIFAR-10 dataset. The example was extracted from [Keras examples](https://github.com/keras-team/keras/blob/master/examples/cifar10_cnn.py)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set up the environment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sagemaker\n",
+    "from sagemaker import get_execution_role\n",
+    "\n",
+    "sagemaker_session = sagemaker.Session()\n",
+    "\n",
+    "role = get_execution_role()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download the CIFAR-10 dataset\n",
+    "Downloading the test and training data will take around 5 minutes."
```

Comment:
Can you add a link to the [cifar datasets page](https://www.cs.toronto.edu/~kriz/cifar.html) as a reference?